azurerm/data_source_logic_app_workflow.go

```diff
@@ -95,7 +96,13 @@ func flattenLogicAppDataSourceWorkflowParameters(input map[string]*logic.Workflo
 
 	for k, v := range input {
 		if v != nil {
-			output[k] = v.Value.(string)
+			// we only support string parameters at this time
+			val, ok := v.Value.(string)
+			if !ok {
+				log.Printf("[DEBUG] Skipping parameter %q since it's not a string", k)
```

Comment:
Should we maybe throw an error here instead of silently ignoring it?

Reply:
this is consistent with the behaviour of the Logic App Workflow resource (and the Template Deployment resource) - which states we only support string values at this time (such that I don't think this is necessary?)